tests/rest/client/v1/test_rooms.py

```diff
@@ -2074,3 +2074,28 @@ def test_create_room(self):
         # Both users should be in the room.
         users = self.get_success(self.store.get_users_in_room(room_id))
         self.assertCountEqual(users, ["@banned:test", "@otheruser:test"])
+
+    def test_message(self):
```

Comment:
Probably need a couple more tests:

* Upgrading a room.

Not sure what else!

Reply:
I added a test for upgrading here.